.circleci/src/jobs/check.yml

```diff
@@ -25,8 +25,12 @@ steps:
       command: yarn workspaces run ts:check
   - unit_test:
       directory: api-client
-  - unit_test:
-      directory: native
+  # TODO IGAPP-1070: Use unit_test command again with coverage
+  - run:
+      name: Unit Tests with Coverage
+      command: yarn workspace native run test:ci --maxWorkers ${TOTAL_CPUS}
+  - store_test_results:
+      path: << parameters.directory >>/reports/unit-test
```

Comment:
I was not able to get jest to work with the `--coverage` flag. Seems like it produces invalid code somehow.

Reply:
Edit: Still failed in the Ci, just disabled the tests now. Created IGAPP-1070 to reenable.

Reply:
Edit: Still failed in the CI, just deleted the tests now. @sarahsporck had a look as well and sadly we both didn't manage to fix it.